dlc_developer_config.toml

```diff
@@ -1,6 +1,6 @@
 [dev]
 # Set to "huggingface", for example, if you are a huggingface developer. Default is ""
-partner_developer = ""
+partner_developer = "huggingface"
```

Comment:
please revert the `dlc_developer_config` changes before merging.

Reply:
Hi @Phyrol, if I understand correctly, we will need to get the PR approved before the revert.